### PR TITLE
Move experimental features to separate folder

### PR DESCRIPTION
**Pull request by tensorchen**
See #133

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 /target
 **/*.rs.bk
 Cargo.lock
+
+# IDEA
```
Comment (jtescher) [inline .gitignore:5]: Not sure we want to add these for each editor, may add this to a global gitignore on your machine?

**Comment by jtescher** [comment]
Thanks looks good! just one small nit

**Comment by tensorchen** [comment]
@jtescher Thanks for your CR, I have cleaned up .gitignore, but CI failure confuses me.

**Comment by jtescher** [comment]
Yeah looks good, the ci issues around nightly should be fixed once https://github.com/open-telemetry/opentelemetry-rust/pull/120 is merged

**Pull request merged**